stories/index.stories.js

```diff
@@ -323,7 +318,7 @@ function renderTilesList(containerEl, list) {
   });
 }
 
-async function setAuthTokenWithOAuthCredentials () {
+async function setAuthTokenWithOAuthCredentials() {
```

Comment:
`setAuthTokenWithOAuthCredentials` and `renderTilesList` should probably be extracted as they are being copied into each story. feel free to ignore.

Reply:
Thanks. I moved them into a separate file and imported them.